monitoring-dashboard/components/org.wso2.ei.dashboard.bootstrap/src/main/java/org/wso2/ei/dashboard/bootstrap/Bootstrap.java

```diff
@@ -105,7 +105,7 @@ public static void main(String[] args) {
 
     private static void startServerWithConfigs() {
 
-        String dashboardHome = System.getenv(DASHBOARD_HOME);
+        String dashboardHome = System.getProperty(DASHBOARD_HOME);
```

Comment:
shall we move this as the class level var and re-use

Reply:
Moved to class level variable.